tests/dev/script-sensitivity-analysis.R

```diff
@@ -0,0 +1,19 @@
+library(ospsuite)
+
+sim <- loadSimulation("C:/projects/OSPSuite-R/tests/data/S1.pkml", loadFromCache = FALSE)
+
+outputSelections <- sim$outputSelections
+
+sensitivity <- SensitivityAnalysis$new(sim)

```

Comment:
Example of usage. Create a new sensitivity for a simulation. If no parameters are defined, all constant parameter that are USED in the simulation will be used for sensitivity